tests/src/cs/eng1/tests/KeyboardInputTests.java

```diff
@@ -0,0 +1,79 @@
+package cs.eng1.tests;
+
+import static org.junit.Assert.*;
+
+import com.badlogic.gdx.Input.Keys;
+import cs.eng1.piazzapanic.utility.KeyboardInput;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(GdxTestRunner.class)
+public class KeyboardInputTests {
+
+    KeyboardInput kbInput = new KeyboardInput();
+
+    @Test
+    public void nullTest() {
+        assertFalse(
+            "Keys other than ones permitted should be unprocessed.",
+            kbInput.keyDown(Keys.ANY_KEY) && kbInput.keyUp(Keys.ANY_KEY)
+        );
+    }
+
+    @Test
+    public void variablesTest() {
+        assertNotNull(
```

Comment:
These variables are of a primitive type `boolean`, which cannot be null. This test will therefore always pass.

Reply:
It might be worth removing the test altogether or coming up with an alternative

Reply:
I've changed this to assertFalse. I was thinking I could test if they existed, but I could just check they start as the right value